mqtt/mqtt-edgehub/src/topic/translation.rs

```diff
@@ -24,7 +24,7 @@ pub fn translate_incoming_subscribe(
     mut subscribe: proto::Subscribe,
 ) -> proto::Subscribe {
     for mut sub_to in &mut subscribe.subscribe_to {
-        if let Some(new_topic) = TRANSLATE_C2D.to_new_topic(&sub_to.topic_filter, client_id) {
+        if let Some(new_topic) = TRANSLATE_C2D.to_internal(&sub_to.topic_filter, client_id) {
```

Comment:
Will this to_internal / to_external be valid when we support the bridge scenario?

Reply:
My hope with new topics we don't need to convert anything